wondrous-app/components/OnboardingDao/DaoCategory/component.tsx

```diff
@@ -0,0 +1,67 @@
+import { Radio, useRadioGroup } from '@mui/material';
+import { ComponentFieldWrapper, Error, FieldInput, FieldLabel, FieldWrapper } from 'components/OnboardingDao/styles';
+import { useField } from 'formik';
+import { CategoriesWrapper, Divider, Label } from './styles';
+
+const CATEGORIES = [
+  '🌎 Social good',
+  '🎬 Media & content',
+  '🐒 NFT collective',
```

Comment:
should this be nft collection?

Reply:
let's match backend here

Reply:
class OrgCategory(SimpleEnum):
    SOCIAL_GOOD = 'social_good'
    MEDIA_CONENT = 'media_content'
    NFT_COLLECTIVE = 'nft_collective'
    INVESTMENT = 'investment'
    DEFI = 'defi'
    SOCIAL = 'social'
    SERVICE_DAO = 'service_dao'
    THINK_TANK = 'think_tank'
    FUN_MEMEABLE = 'fun_memeable'
    BUILDING_PRODUCTS = 'building_products'

Reply:
Thanks. I'll update.

Reply:
@terryli0095 How are we going to handle custom goals?
<img width="316" alt="image" src="https://user-images.githubusercontent.com/8164667/181491792-5e673083-d2ed-4d87-ae14-57a0184af8f6.png">